types/trusted-types/tslint.json

```
@@ -1 +1,6 @@
-{ "extends": "dtslint/dt.json" }
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "interface-over-type-literal": false
+    }
+}
```

```suggestion
{ "extends": "dtslint/dt.json" }
```
if you accept a comment about L.48 already showing interfaces are used to define methods, please accept that suggestion

---

Done.